ppx/ppx.ml

```diff
@@ -653,13 +653,13 @@ let rec recursivelyTransformNamedArgsForMake mapper ctxt expr list =
             None
       in
       recursivelyTransformNamedArgsForMake mapper ctxt expression
-        ( ( Str_label.of_arg_label arg
```

Comment:
I think this is what made ci fail, did you install latest `ocamlformat` with `opam install ocamlformat`?

Reply:
Yeah that was my bad, I was using an outdated version of `ocamlformat` -- should be fixed now :)